suites/apis/submitFileTest.js

```diff
@@ -5,20 +5,11 @@ let chaiAsPromised = require('chai-as-promised');
 chai.use(chaiAsPromised);
 let expect = chai.expect;
 
-
-Feature('SubmitFileTest');
+let util = require('./utilApis');
 
 
-const clone = function(obj) {
-  return JSON.parse(JSON.stringify(obj));
-};
+Feature('SubmitFileTest');
 
-const extractFile = function(nodes) {
-  let file_node_name = Object.keys(nodes).find((node_name) => { return nodes[node_name].category === 'data_file' });
-  let bfd = clone(nodes[file_node_name].data);
-  delete nodes[file_node_name];
-  return bfd;
-};
 
 // list for non data_file nodes
 let nodes, nodes_list;
```

Comment:
same here, where is the value for those variables?

Reply:
https://github.com/uc-cdis/gen3-qa/blob/feat/pidgin-tests/suites/apis/submitFileTest.js#L135-L140